packages/web-components/src/components/markdown/markdown.ts

```diff
@@ -55,14 +59,16 @@ class C4DMarkdown extends LitElement {
       link(href, title, text) {
         return `<${prefix}-link href="${href}" size="lg" ${
           title ? `title="${title}"` : ''
-        }>${text}</${prefix}-link>`;
+        } part="link">${text}</${prefix}-link>`;
       },
       list(body, ordered) {
         const tag = `${prefix}-${ordered ? 'ordered' : 'unordered'}-list`;
-        return `<${tag}>${body}</${tag}>`;
+        return `<${tag} part="${
+          ordered ? 'ordered-list' : 'unordered-list'
```

Comment:
Lets add a consistent name to either case so if someone wanted to target regardless of the kind of list, they could.
```suggestion
          ordered ? 'list ordered-list' : 'list unordered-list'
```